src/components/modals/NightOrderModal.vue

```diff
@@ -154,7 +154,7 @@ export default {
 <style lang="scss" scoped>
 @import "../../vars.scss";
 
-.toggle {
+.toggle-reference {
```

Comment:
The style tag (L154) is scoped, so there's no need to choose a unique name for this class.

Reply:
Indeed its not needed, just style.